### ISSUE TITLE
Determine goals of the game

### ISSUE THREAD

**Issue by ClawsofShadow** [ClawsofShadow/platformer-tutorial#1]
# What is the goal of your game? 

###  Goal of the game is to get a sword to kill the Boss.

# What is the goal of each level/world?

Level 4 last level will be a Boss to fight.

The first 3 levels will have a mini Boss, a flag and a door.

You can do level 1 and 2 in any order you like, but not for level 3 and 4.

# How does the player win?

On levels 1-3 you will work to unlock a sword to kill the Boss with.
Once you kill the main Boss you win the whole game. For the main boss the player will get locked in the room.

# How does the player lose?

Player will have three lives, when you lose all three the player restarts the game.

Lives are lost by dying from enemies or falling to your death.

# Name a core mechanic and how it will work?

On level 2 the character will finish getting supplies to make a sword.

The character will have to kill a mini Boss to get the item for that level to craft the sword.

Killing the mini Boss is not how the player will when that level, he is able to skip the mini Boss but he will not be able to kill the main Boss. Until he has a sword the game won't let him go on to 4. The player will not get locked in the room with the any of the mini boss. 

The player can kill boss's, and the walking enemy with the sword but nothing else.

Near the end of level 3 the character will craft his sword.
basically the player puts the stuff for the sword in a thing that puts it together for him. 

The sword will break over time and is used to kill enemies but it is mostly use.
If the sword breaks the player can redo levels 1-3 to get it back.

The mini boss in level 1 will give you a handle.
Mini boss in level 1 will be shooting fire balls at you.
Each fire ball will take one heart.

The mini boss in level 2 will give you a blade for the sword.
Mini boss in level 2 will be throwing fire sword at you.
Each sword will take two hearts.

**Comment by tkrause** [comment]
Boss not Bose

What are levels 1-8? You won't have time to make 11 levels this year. Just do 1-3 getting parts for the sword, level 4 killing the final boss.

Explain which parts you get from each mini boss it sounds like a handle, blade, and hand guard or something?

**Comment by tkrause** [comment]
You also need to create issues for implementing a sword, a boss (does the player get stuck in room/scene with the boss), any additional mini bosses, any additional enemies.

Are you able to do the levels in any order you want or is it a set order? How's the player combine the parts to make the sword? Or does it just happen after level 3?

You say the player can kill enemies with the sword but it breaks? How do you get it back then? What enemies can you kill? Is it everything? Spikes included? That seems broken.
